package/base-files/files/lib/upgrade/nand.sh

```diff
@@ -10,8 +10,13 @@ CI_KERNPART="${CI_KERNPART:-kernel}"
 # 'ubi' partition on NAND contains UBI
 CI_UBIPART="${CI_UBIPART:-ubi}"
 
+# <<<<<<<
 # 'rootfs' partition on NAND contains the rootfs
 CI_ROOTPART="${CI_ROOTPART:-rootfs}"
+# =======
+# VID header offset is used e.g. to ignore sub pages by setting them to 2048
+CU_UBI_VID_HEADER_OFFSET=""
+# >>>>>>>
```

Comment:
This looks like a merge conflict

Reply:
I also wondered about this, because the merge conflict indicators are prefixed by hashs (i.e. commenting them out).